librz/include/rz_bin.h

```diff
@@ -936,8 +936,10 @@ RZ_API const RzList *rz_bin_object_get_strings(RZ_NONNULL RzBinObject *obj);
 RZ_API const RzList *rz_bin_object_get_mem(RZ_NONNULL RzBinObject *obj);
 RZ_API const RzList *rz_bin_object_get_resources(RZ_NONNULL RzBinObject *obj);
 RZ_API const RzList *rz_bin_object_get_symbols(RZ_NONNULL RzBinObject *obj);
-RZ_API const RzList *rz_bin_object_reset_strings(RZ_NONNULL RzBin *bin, RZ_NONNULL RzBinFile *bf, RZ_NONNULL RzBinObject *obj);
-RZ_API bool rz_bin_object_is_string(RZ_NONNULL RzBinObject *obj, ut64 va);
+RZ_API bool rz_bin_object_reset_strings(RZ_NONNULL RzBin *bin, RZ_NONNULL RzBinFile *bf, RZ_NONNULL RzBinObject *obj);
+RZ_API RzBinString *rz_bin_object_string_at(RZ_NONNULL RzBinObject *obj, ut64 address, bool is_va);
```

Comment:
I would call this `rz_bin_object_get_string_at` to be consistent with the other functions around